knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml

```diff
@@ -875,6 +875,7 @@ rules:
       - "*"
     resources:
       - configmaps
+      - services
```

Comment:
@pierDipi is this already in midstream?

I think we should do that with a patch, instead of just hard coding in this file

Reply:
Yes, it's now in midstream

Reply:
actually, it's not yet https://github.com/openshift-knative/eventing-kafka-broker/pull/226

Reply:
yeah, sad :disappointed: